src/include/common/task_system/task.h

```diff
@@ -21,7 +23,7 @@ using lock_t = std::unique_lock<std::mutex>;
  * finalizeIfNecessary, users should save it somewhere that can be accessed in
  * finalizeIfNecessary(). See ProcessorTask for an example of this.
  */
-class Task {
+class KUZU_API Task {
```

Comment:
I needed to do this to make Louvain work because these classes are used by `gds_utils_mem.cpp`, which I moved to `extension/gds` because that class is only used by Louvain and seems to be optimized out of `libkuzu` otherwise.

Is this okay to export.

Alternatively, I'm unclear how the other classes in `function/gds` continue to work.

Reply:
yes, this is ok to export